packages/components/src/components/card/card.e2e.ts

```diff
@@ -8,3 +8,4 @@ describe('t-card', () => {
     expect(element).toHaveClass('hydrated');
   });
 });
+// Test für bild, alt-text
```

Comment:
Please use English for comments ;)

Reply:
it's okay. this comment is only for me :)

Reply:
If it's merged, it's for everyone